typescript/scripting-api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wot-typescript-definitions",
-  "version": "0.8.0-SNAPSHOT.16",
+  "version": "0.8.0-SNAPSHOT.17",
```

Comment:
should be "0.8.0-SNAPSHOT.18"

0.8.0-SNAPSHOT.17 is published already, see https://www.npmjs.com/package/wot-typescript-definitions

Reply:
Thank you, is now 18 :)